### PR TITLE
Jc 1072 2 Edit Connectors description text

### PR DESCRIPTION
**Pull request by MelissaFlinn**
Edit Connectors descriptions
Edit dead letter queue description

### CODE REVIEW

File: cos-fleet-catalog-connectors/azure/azure-storage-blob-0.1/pom.xml [hunk 1]
```diff
@@ -55,7 +55,7 @@
                         <connector>
                             <name>${cos.connector.type}-sink-${cos.connector.version}</name>
                             <title>${project.name} sink</title>
-                            <description>${project.description} sink</description>
+                            <description>Send data to Azure Blob storage.</</description>
```
Comment (rinaldodev) [inline cos-fleet-catalog-connectors/azure/azure-storage-blob-0.1/pom.xml:58]: There is an extra `</` in here

**Review by rinaldodev — CHANGES_REQUESTED** [review]
Thanks for the very organized PR. I found only an XML syntax mistype that needs to be fixed.

**Comment by MelissaFlinn** [comment]
Thanks for your review - I fixed that typo.

File: cos-fleet-catalog-connectors/aws/aws-sns-0.1/pom.xml [hunk 2]
```diff
@@ -36,7 +36,7 @@
                             <description>${project.description} sink</description>
                             <adapter>
                                 <prefix>aws</prefix>
-                                <name>${cos.connector.type}-sink</name>
+                                <name>Send data to an Amazon Simple Notification Service (SNS) topic.</name>
```
Comment (rinaldodev) [inline cos-fleet-catalog-connectors/aws/aws-sns-0.1/pom.xml:39]: This was changed in a different place. Should be in <description> a few lines above.

**Review by rinaldodev — CHANGES_REQUESTED** [review]
Sorry, there is one more fix to be made. Didn't catch it before.

File: cos-fleet-catalog-connectors/aws/aws-sns-0.1/pom.xml (hunk 2 above)
Comment (MelissaFlinn) [reply cos-fleet-catalog-connectors/aws/aws-sns-0.1/pom.xml]: Sorry about that - thank you for catching it

**Comment by MelissaFlinn** [comment]
@rinaldodev I fixed the second error - thanks

**Pull request merged**